src/redux/books/booksSlice.js

```diff
@@ -1,11 +1,54 @@
-import { createSlice } from '@reduxjs/toolkit';
-import booksList from '../../booksList';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+// import booksList from '../../booksList';
```

Comment:
- [Optional] Please consider removing any piece of commented code from your code base, commented code makes your project look `unprofessional` and is considered a sign of `code` that is `not clean`.